### PR TITLE
[WIP]don't list modules from other releases

### PR DESCRIPTION
**Pull request by andreeap**
This is a try at https://github.com/CPAN-API/cpan-api/issues/125. I solved the problem of showing the modules from other releases, but in the search page, it's still wrong and I am not sure where that comes from. Maybe that is still from the api?

### CODE REVIEW

File: root/release.html [hunk 1]
```diff
@@ -43,7 +43,15 @@
...
 FOREACH file IN files;
   IF file.documentation && file.module;
-    modules.push(file);
+    found = 0;
+    FOREACH module IN file.module;
+      IF module.name != file.documentation;
+        found = found + 1;
+        IF found == 1;
+          modules.push(file);
+        END;
```
Comment (oalders) [inline root/release.html]: You could shorten this to:

```
IF module.name != file.documentation;
    modules.push(file);
END;
```

**Comment by oalders** [comment]
Could you summarize how this fixes the problem?

**Comment by andreeap** [comment]
The way it was before it added modules from another release that were listed as a documentation, but by checking just the ```IF module.name != file.documentation;``` and then add the file it will push the same module 3 times and this is why I did that little thing with found. Of course, maybe it's something that I don't understand and it could be easily solved in some other way.

**Comment by oalders** [comment]
@rwstauner thoughts?

**Comment by oalders** [comment]
@andreeap is this still a WIP?

**Comment by oalders** [comment]
@andreeap ping?

**Comment by rwstauner** [comment]
It took me a while to figure out what this actually doing.
This is not the right way to fix this.
There's a bug in the template logic (before this change) so that this new code just isn't being run,
which happens to make the undesired modules disappear.

I'm considering several other (much simpler changes) that will change the way things are displayed and likely spark more discussion about how this should really be handled.

**Comment by rwstauner** [comment]
Fixing these two (similar) template bugs made a significant difference in how those modules are displayed for JSON and JSON-PP:
cd0b4d69f421722dc2ac2d03e29743dd88db1409
77ba2e6c66461b634703d32d96b88249e13e3f5f

**Comment by rwstauner** [comment]
Thanks for your effort on this, it did help to identify and fix the overall issue!
